code/game/machinery/recycler.dm

```diff
@@ -42,9 +42,9 @@
 
 /obj/machinery/recycler/examine(mob/user)
 	. = ..()
-	. += "The power light is [(stat & NOPOWER) ? "off" : "on"]."
-	. += "The safety-mode light is [safety_mode ? "on" : "off"]."
-	. += "The safety-sensors status light is [emagged ? "off" : "on"]."
+	. += "<span class='notice'>The power light is <b>[(stat & NOPOWER) ? "<b>off</b>" : "<b>on</b>"]</b>."
+	. += "The operation light is [emergency_mode ? "<b>off</b>. \The [src] has detected a forbidden object with its sensors, and has shut down temporarily." : "<b>on</b>. \The [src] is active."]"
```

Comment:
Some superfluous `<b>` tag removal. Also you don't need to use `\The`. simply writting `[src]` will append a "the" to the front.
```suggestion
	. += "<span class='notice'>The power light is [(stat & NOPOWER) ? "<b>off</b>" : "<b>on</b>"]."
	. += "The operation light is [emergency_mode ? "<b>off</b>. [src] has detected a forbidden object with its sensors, and has shut down temporarily." : "<b>on</b>. [src] is active."]"
```